pgsync/sync.py

```diff
@@ -442,14 +442,14 @@ def should_skip_event(self, event: dict) -> bool:
             logger.debug(f"should_skip_event: skipping event, event[\"old\"] and event[\"new\"] is set to None and tg_op is {event['tg_op']}")
             return True
 
-        if event['indices'] is not None:
+        if event['indices'] != type(None) and event['indices'] is not None:
             # event['indices'] set to None when pgsync parsing replication slot
             if self.index in [event['indices']]:
                 logger.debug(f"should_skip_event: skipping event, none of the event's index names \"{event['indices']}\" matches pgsync JSON schema name")
                 return True
 
         if (event['schema'], event['table']) in self._schema_fields:
-            if CHANGED_FIELDS in event and event[CHANGED_FIELDS] is not None:
+            if CHANGED_FIELDS in event and event[CHANGED_FIELDS] != type(None) and event[CHANGED_FIELDS] is not None:
```

Comment:
```python
if CHANGED_FIELDS in event and event[CHANGED_FIELDS] != type(None) and event[CHANGED_FIELDS] is not None:
```
is unnecessary, and not very readable
this does the same thing, but readable:
```python
if event.get(CHANGED_FIELDS) is not None:
```

Reply:
yes, and "`.get`" is called on a dict type:

```python
>>> event = {"foo": 42, "change_events": {"a": 42}}
>>>
>>> event.get("change_events")
{'a': 42}
>>>
>>> event.get("no change events")
>>>
>>> event.get("no change events") == None
True
```yes, and "`.get`" is called on a dict type:

```python
>>> event = {"foo": 42, "change_events": {"a": 42}}
>>>
>>> event.get("change_events")
{'a': 42}
>>>
>>> event.get("no change events")
>>>
>>> event.get("no change events") == None
True
```

Reply:
yes it may be, but it 2.2-3 times slower then using event[foo_var]

Reply:
the idea is not to micro optimize, but to have something that we can maintain and understand better:

```python
if event.get(CHANGED_FIELDS) is not None:
```

is clear

```python
if CHANGED_FIELDS in event and event[CHANGED_FIELDS] != type(None) and event[CHANGED_FIELDS] is not None:
```

is not, and also all together is slower than a single "`event.get`"

Reply:
This is not a micro optimization, this function is ran in the thread that read notification channel - should_skip_event - push to redis, poll_db. When migrations happened, this thread is about 5 times more CPU than allocated to whole pgsync 

1. from notify channel pgsync can receive None value
    if indices is set to none, we skip event and not process this event  
2. from parsing replication slot pgsync receives NoneType for fields indices, change_orders ...
    if indices is set to NoneTyep  pgsync should process event